### ISSUE TITLE
Bug: formatter.rb:3700:in `block in adjust_other_alignments': undefined method `[]' for nil:NilClass (NoMethodError)

### ISSUE THREAD

**Issue by ivoanjo** [ruby-formatter/rufo#87]
While looking into #83 I tried rufo on a number of repositories for commonly-used gems and came across a few issues.

This one comes from the [i18n](https://github.com/svenfuchs/i18n) gem.

Traceback:

```
Traceback (most recent call last):
        18: from .rvm/gems/ruby-2.5.0/bin/ruby_executable_hooks:15:in `<main>'
        17: from .rvm/gems/ruby-2.5.0/bin/ruby_executable_hooks:15:in `eval'
        16: from .rvm/gems/ruby-2.5.0/bin/rufo:23:in `<main>'
        15: from .rvm/gems/ruby-2.5.0/bin/rufo:23:in `load'
        14: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/exe/rufo:4:in `<top (required)>'
        13: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:12:in `run'
        12: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:40:in `run'
        11: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:81:in `format_args'
        10: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:81:in `each'
         9: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:82:in `block in format_args'
         8: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:124:in `format_file'
         7: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/command.rb:165:in `format'
         6: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:171:in `format'
         5: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3600:in `dedent_calls'
         4: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3600:in `each'
         3: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3612:in `block in dedent_calls'
         2: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3696:in `adjust_other_alignments'
         1: from .rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3696:in `each'
.rvm/gems/ruby-2.5.0/gems/rufo-0.2.0/lib/rufo/formatter.rb:3700:in `block in adjust_other_alignments': undefined method `[]' for nil:NilClass (NoMethodError)
```

File that triggers the issue: <https://raw.githubusercontent.com/svenfuchs/i18n/ed3623f1a9a8eedd968ac5fd7040faccbb171547/lib/i18n/exceptions.rb>

**Comment by gingermusketeer** [comment]
Thanks for reporting this. I have a failing test on a branch for this issue https://github.com/ruby-formatter/rufo/compare/chore/fix-i18n-gem-formatting?expand=1

**Comment by mjago** [comment]
Fixed with #112
